recipes/monet/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "monet" %}
+{% set version = "2.1.5" %}
+
+package:
+  name: monet
+  version: {{ version }}
+
+source:
+  #git_url: https://github.com/noaa-oar-arl/MONET.git
+  #git_tag: develop
+  url: https://github.com/noaa-oar-arl/MONET/archive/v2.2.0.tar.gz
+  sha256: 293ca8ca167c77843a165e6032f18ea50b96f964264087ce15dabde4bf0a270e
+  #url: https://github.com/noaa-oar-arl/MONET/archive/v2.1.3-dev.2.tar.gz
+  # sha256: 7027a0478cab0f877ebc7e412a0249d0578c0944fab4da6f341175995b957931
+
+build:
+  number: 0
+  #noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - python >=3.6
+  host:
+    - python
+    - pip
+  run:
+    - git
+    - python
+    - numpy
+    - pandas
+    - xarray
+    - dask
+    - cartopy
+    - matplotlib
+    - seaborn
+    - pyresample
+    - python-stratify
+    - pydecorate
+    - xesmf  # [not win]
+#    - global_land_mask # [not win]
+    
+test:
+  imports:
+    - monet
+    
+about:
+  home: https://github.com/noaa-oar-arl/monet
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: The Model and ObservatioN Evaluation Tool - for meteorology and AQ
+  doc_url: https://monet-arl.readthedocs.io
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```

Comment:
delete comments.

Reply:
deleted

Reply:
It looks like the comments are still there under the `recipe-maintainers` section. I think that's the last item and then I can merge this.